src/node_api.cc

```diff
@@ -928,6 +928,10 @@ napi_status napi_create_external_buffer(napi_env env,
   NAPI_PREAMBLE(env);
   CHECK_ARG(env, result);
 
+#if defined(V8_ENABLE_SANDBOX)
```

Comment:
I don't find this definition on v14.x. Maybe we can simply remove this change?

Reply:
That could make sense, I think we wanted the backport so that people could compile on the oldest LTS and use the value for the error code.  

They question is whether it's better to keep it closer to the original PR even if the define will never be set or to remove it?

Reply:
@richardlau do you have an opinion from the Releaser perspective?  I'm happy either way.  We can remove the define or leave it in to make it a "backport" versus a "backport" with modifications whichever is easier/preferrable for getting it landed.

Reply:
@mhdawson No preference either way. I think it's very unlikely `V8_ENABLE_SANDBOX` would be introduced to Node.js 14 if it isn't already there given that Node.js 14 is in maintenance and will go End-of-Life in April.

cc @nodejs/lts

Reply:
@legendecas since @richardlau figures there is no preference from the Releasers perspective I don't don't feel strongly either. I'm happy to leave as it as I don't think it hurts anything but also happy to remove that i you think that's best.

Reply:
I'm fine with both ways. Thanks for the clarification.